zerver/lib/send_email.py

```diff
@@ -416,23 +416,20 @@ def clear_scheduled_invitation_emails(email: str) -> None:
 
 
 @transaction.atomic(savepoint=False)
-def clear_scheduled_emails(user_ids: List[int], email_type: Optional[int] = None) -> None:
+def clear_scheduled_emails(user_id: int, email_type: Optional[int] = None) -> None:
```

Comment:
I think we used the list construction for management shell usage in the past, but one can always do a loop, so this is fine.